libs/facilities/shared/models/src/lib/facilities-shared-models/facilities-shared-models.component.ts

```diff
@@ -0,0 +1,11 @@
+import { CommonModule } from '@angular/common';
+import { Component } from '@angular/core';
+
+@Component({
+	selector: 'lib-facilities-shared-models',
+	standalone: true,
+	imports: [ CommonModule ],
+	templateUrl: './facilities-shared-models.component.html',
+	styleUrl: './facilities-shared-models.component.css',
+})
+export class FacilitiesSharedModelsComponent {}
```

Comment:
Why does this component exist? No logic included and no real HTML?

Reply:
I dont even know what this lib does in there, i just removed it